Source/WTF/wtf/Vector.h

```diff
@@ -64,7 +64,7 @@ struct VectorDestructor<false, T>
 template<typename T>
 struct VectorDestructor<true, T>
 {
-    static void destruct(T* begin, T* end) 
```

Comment:
Please update your IDE to stop doing this. This is frowned upon as it pollutes the PRs.

Reply:
Removed the whitespace changes ✔️